decorator.rb

```diff
@@ -0,0 +1,30 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, "#{self.class} has no implemented method #{__method__}"
+  end
+end
+
+class Decorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
+
+  def correct_name
+    @nameble.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameble.correct_name.capilize
```

Comment:
## * Kindly fix this typo here ( capilize) instead of **capitalize** and ( nameble) instead of **nameable**  as it throws an error 
![image](https://user-images.githubusercontent.com/52497246/210358554-f2cbd843-5778-4e46-be31-af9f27c1d512.png)